### PR TITLE
Implemented the importing of a project file.

### PR DESCRIPTION
**Pull request by JamesVaughan**
Implements issue #41 the importing of a project file.

### CODE REVIEW

File: src/XTMF2/Controllers/ProjectController.cs [hunk 1]
```diff
@@ -74,6 +74,58 @@ public bool CreateNewProject(User owner, string name, out ProjectSession session
...
+            }
+
+            if (string.IsNullOrWhiteSpace(name))
+            {
+                error = "A project must have a non-blank unique name.";
+                return false;
+            }
+            if (string.IsNullOrWhiteSpace(filePath))
+            {
+                throw new ArgumentException("message", nameof(filePath));
```
Comment (BrendanReilly) [inline src/XTMF2/Controllers/ProjectController.cs]: Should the message be replaced with the information about it being null / whitespace?

**Review by BrendanReilly — CHANGES_REQUESTED** [review]
Everything looks good minus the missing error text (left in review comment).

File: src/XTMF2/Controllers/ProjectController.cs (hunk 1 above)
Comment (JamesVaughan) [reply src/XTMF2/Controllers/ProjectController.cs]: Good catch.

**Review by BrendanReilly — APPROVED** [review]
Looks good after the most recent change.

**Pull request merged**